### DISCUSSION TITLE
About dataset formats

### DISCUSSION THREAD

**Discussion by kin0303** [lifeiteng/valle#25] [Q&A] [answered]
What dataset format is used? Is it same as LJSpeech format?
ID_Speaker|Text1|Text2
or can it only use the libritts dataset?

**Comment by MisakaMikoto96** [comment, marked as answer]
you should customize your `dataset` processing and `dataloader` as lhoste could not work on some other data formats.

**Reply by kin0303** [reply]
> you should customize your `dataset` processing and `dataloader` as lhoste could not work on some other data formats.

Hai @MisakaMikoto96, have you ever done it? I saw the libriTTS format before, and it's very different from the dataset I have. My dataset format is the same as MAILAB or LJSpeech

Can you explain where should I change it? I still don't understand

**Reply by MisakaMikoto96** [reply]
Yes, I rewrite the dataloader and dataset for another language means you just could check your input(text_id sequence/acoustic token) ( you could refer to the input of Libritts )and customize your languange inputs without k2&icefall dependence.

**Reply by kin0303** [reply]
> Yes, I rewrite the dataloader and dataset for another language means you just could check your input(text_id sequence/acoustic token) ( you could refer to the input of Libritts )and customize your languange inputs without k2&icefall dependence.

What language do you use, is it a language other than English?
